src/dndbeyond/content-scripts/character.js

```diff
@@ -883,6 +883,32 @@ async function rollItem(force_display = false, force_to_hit_only = false, force_
             }
         }
 
+        // 2024 brutal strike
+        if (character.getSetting("brutal-strike")) {
+            let strikeDieCount = character.hasClassFeature("Brutal Strike") ? 1 : 0;
+            if (character.hasClassFeature("Improved Brutal Strike")) strikeDieCount += 1;
+
+            let brutal_strike_dmg = `${strikeDieCount}d10`;
+
+            if(((properties["Attack Type"] == "Melee" && ((properties["Properties"].includes("Versatile") && character.getSetting("versatile-choice") != "one") || 
+                properties["Properties"].includes("Two-Handed"))) ||
+                item_name == "Polearm Master - Bonus Attack" ||
+                item_name == "Pole Strike")) {
+                if(character.hasGreatWeaponFighting(2014)) {
+                    brutal_strike_dmg += "ro<=2";
+                } else if(character.hasGreatWeaponFighting(2024)) {
+                    brutal_strike_dmg = brutal_strike_dmg.replace(/([0-9]*)d([0-9]+)([^\s+-]*)(.*)/g, (match, amount, faces, roll_mods, mods) => {
+                        return new Array(parseInt(amount) || 1).fill(`1d${faces}${roll_mods}min3`).join(" + ") + mods;
+                    });
+                }
+            }
```

Comment:
I'm seeing this block of code more than once at this point, this would be better as a method you can call 
```js
brutal_strike_dmg= this.applyGWFModifiers(item_name, properties, brutal_strike_dmg);
```